components/Code/index.jsx

```diff
@@ -8,11 +8,13 @@ const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/synta
 const { canonical: canonicalLanguage } = require('@readme/syntax-highlighter');
 const copy = require('copy-to-clipboard');
 
-function CopyCode({ code, rootClass = 'rdmd-code-copy', className = '' }) {
+function CopyCode({ codeRef, rootClass = 'rdmd-code-copy', className = '' }) {
   const copyClass = `${rootClass}_copied`;
   const button = React.createRef();
   /* istanbul ignore next */
   const copier = () => {
+    const code = codeRef.current.textContent;
```

Comment:
Slick.

Reply:
Let me double check. I tested before writing the tests and stuff.

Reply:
Ok, after much linkage, it works for me locally.